src/components/Tag/Tag.scss

```diff
@@ -20,17 +20,65 @@ $icon-size: rem(16px);
     color: var(--p-text-disabled, color('ink', 'lightest'));
   }
 
-  .TagText {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-  }
+  &.clickable {
+    @include unstyled-button;
+    cursor: pointer;
+    padding: 0 spacing(tight);
+    background-color: color('sky');
+
+    // Start - Delete below me during newDesignLanguage rollout
```

Comment:
Are these comments still relevant? I feel like they can be removed

Reply:
Yes, for the buttons we have to do the design language stuff within the `.newDesignLanguage` class (`rule: var(--p-variable-name)`) instead of using the fallback parameter of `var` (`rule: var(--p-variable-name), fallbackColorFunc('sky', 'dark')`)